cmd/containerd-shim-runc-v2/task/service.go

```diff
@@ -115,9 +115,11 @@ type service struct {
 
 	containers map[string]*runc.Container
 
-	lifecycleMu  sync.Mutex
-	running      map[int][]containerProcess // pid -> running process, guarded by lifecycleMu
-	pendingExecs map[*runc.Container]int    // container -> num pending execs, guarded by lifecycleMu
+	lifecycleMu   sync.Mutex
+	running       map[int][]containerProcess // pid -> running process, guarded by lifecycleMu
+	pendingExecs  map[*runc.Container]int    // container -> num pending execs, guarded by lifecycleMu
+	skipInitExit  containerProcess
+	skipInitEvent runcC.Exit
```

Comment:
Do these need to be members of the struct?  They appear to only be accessed from within `processExits` and could be local variables of that function.